src/AutoRest.CSharp.V3/Output/Builders/ClientBuilder.cs

```diff
@@ -34,31 +34,49 @@ public static Client BuildClient(OperationGroup operationGroup)
             }
 
             string clientName = operationGroup.CSharpName();
-            List<Method> methods = new List<Method>();
-            List<Paging> pagingMethods = new List<Paging>();
+            Dictionary<string, (Operation Operation, Method Method)> operationMethods = new Dictionary<string, (Operation, Method)>();
             foreach (Operation operation in operationGroup.Operations)
             {
                 Method? method = BuildMethod(operation, clientName, clientParameters);
                 if (method != null)
                 {
-                    methods.Add(method);
-                    var pageable = operation.Extensions.GetValue<IDictionary<object, object>>("x-ms-pageable");
-                    if (pageable != null)
+                    operationMethods.Add(operation.Language.Default.Name, (operation, method));
+                }
+            }
+
+            List<Method> nextPageMethods = new List<Method>();
+            List<Paging> pagingMethods = new List<Paging>();
+            foreach ((string _, (Operation operation, Method method)) in operationMethods)
```

Comment:
Lets add a private struct for `(Operation operation, Method method)`. Right now it makes code very verbose.

Reply:
Done.